platform-designs/design-architecture/README.md

```diff
@@ -12,7 +12,9 @@ IA focuses on two elements: **where a user is** and **where to find information
 The diagram reads as a hierarchy from top to bottom with each layer starting with a "parent" element that has corresponding "child" elements. 
 
 ### IA Diagram
-![AutSPACEs IA Diagram 15-11-2021](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-architecture/autspaces-ia-diagram-15-11-2021.md)
+* Updated 21-02-2022: [Information Architecture Diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=XndytNqcHWh8WrCRgyDL&title=IA_AutSPACEs.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1THiziivGBr89J5x9oJahXuHtL7iib8J1%26export%3Ddownload)
+* Previous 15-11-2021
+![AutSPACEs IA Diagram 15-11-2021](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-architecture/autspaces-ia-diagram-15-11-2021.png)
```

Comment:
Great work managing the changelog and making both viewable - it'll be super helpful when understanding when things were changed. 

We can always use github's diff/roll back features to understand the different versions, but I also like how you have make specific note of it here => Happy to keep this style going forward.